### PR TITLE
Set verbose logging level for glocaltokens

### PR DESCRIPTION
**Pull request by leikoilja**
Setting verbose logging level for glocaltokens based on integration logging level.

Also prepare for release by bumping up the version

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -24,10 +24,12 @@ mypy = "^0.812"
 # Reasons disabled:
 # too-many-* - not enforced for the sake of readability
 # too-few-* - same as too-many-*
+# unexpected-keyword-arg - argues that glocaltokens has no verbose argument
```
Comment (KapJI) [inline pyproject.toml:27]: It's a valid complain. You need to update `glocaltokens` version in poetry as well.

Comment (leikoilja) [reply pyproject.toml:27]: yeh, it felt like i m missing something here :D

Comment (leikoilja) [reply pyproject.toml]: Fixed ✅

**Comment by KapJI** [comment]
Hm. I think version constraint in `pyproject.toml` also need to be changed. Right now it's `0.2.4 or above` which is not right. Try `poetry add glocaltokens==0.2.6`.

**Comment by leikoilja** [comment]
Hm, that's weird. I have ran `poetry update glocaltokens` I would have expected it would update the version in `pyproject.toml` file as well. Needed to run `remove` and `add glocaltokens` to reinstall it to the newest version 🤔

**Comment by KapJI** [comment]
Yeah, `poetry update` only updates version in `poetry.lock` using current constraints from `pyproject.toml`.
So right now it's `0.2.6 or above` which is good but let's maybe keep constraint in sync with `manifest.json` and set fixed version here as well?

**Comment by leikoilja** [comment]
Hmmmm, sounds good with me :)
Fixed

**Review by KapJI — APPROVED** [review]
Thanks! 🎉

**Pull request merged**